examples/notebooks/discrete_choice_example.ipynb

```diff
@@ -99,9 +96,7 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "affair_mod = logit(\"affair ~ occupation + educ + occupation_husb\" \n",
```

Comment:
Can you trim trailing whitespace while in the neighborhood

Reply:
Thanks for pointing out. Fixed.